src/main/java/core/service/OperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package core.service;
+
+import core.db.FruitTransaction;
+import core.db.StorageService;
+
+public interface OperationHandler {
+    void addTransaction(FruitTransaction transaction,
+                        StorageService<FruitTransaction> storageService);
+}
```

Comment:
```suggestion
    void addTransaction(FruitTransaction transaction);
}
```
pass the second parameter using constructors of handlers